### PR TITLE
Use the branch sdk to generate deep links.

### PR DESCRIPTION
**Pull request by telaviv**
The code this was based on only generated the main "clickUrl" with the branch sdk, and then statically generated the others. I kept that behavior because I wasn't sure if this was intentional. 

👓 : @prashtx @uzi

### CODE REVIEW

**Comment by prashtx** [comment]
Looks like there are some build issues. Let's discuss the use of static vs dynamic URLs IRL.

💅

**Comment by telaviv** [comment]
💇

**Comment by telaviv** [comment]
💇  fixed linting issues.

File: src/lib/branch.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+// Branch is the api we use to generate "deep links". These are links to
+// our mobile app. With each link generated we also attach meta data.
+// This meta data can then be used to analyze download patterns of our users.
+
+
+// Get loid values either from the account state or the cookies.
+function getLoidValues(accounts) {
+  if (accounts.me) {
+    return {
```
Comment (uzi) [inline src/lib/branch.js]: Could also do

```js
const {loid, loidCreated} = accounts.me;
return {
  loid,
  loidCreated,
};
```

File: src/lib/branch.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      loid: accounts.me.loid,
+      loidCreated: accounts.me.loidCreated,
+    };
+  }
+
+  const loid = cookies.get('loid');
+  const loidCreated = cookies.get('loidcreated');
```
Comment (uzi) [inline src/lib/branch.js]: Hmm... and you can fall through to this if there's no loid or loidCreated and just share the return.

**Comment by uzi** [comment]
Yeah, looks pretty straightforward to me... maybe just a little more cleanup?

**Comment by telaviv** [comment]
💇

**Comment by uzi** [comment]
:fish:

**Pull request merged**